bin/server

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/env node
 'use strict';
 
+const Errors = require('../lib/errors');
+const SUPPORTED_ALGORITHMS = ['SHA256', 'SHA512'];
+
 require('../lib/config');
+if (['SHA256', 'SHA512'].filter((a) => a === Config.get('local:algorithm').toUpperCase()).length === 0) {
```

Comment:
Okay. We should probably use `SUPPORTED_ALGORITHMS` in the if check, so the list in the error message matches what's checked.